docs/index.rst

```diff
@@ -26,9 +26,9 @@ Documentation
     :maxdepth: 2
 
     general/index
+    usage/index
     guidelines/index
     structure/index
     testing/index
     codestyle/index
-    tips/index
```

Comment:
I'd still keep them separate as we will get general frontend tips inside there too